### DISCUSSION TITLE
Associate 2 z-wave shutter modules

### DISCUSSION THREAD

**Discussion by S1M8N** [zwave-js/zwave-js#3969] [Q&A] [answered]
Hello everyone,

I use two z-wave roller shutter modules to control two rollers shutters. However, physically I only have one button. I can therefore control a roller shutter physically. Is it possible to associate my two z-waves modules together to make one?

The first z-wave module is the MCO Home **MH-C421** (this one has the physical button) and the second module is the Fibaro roller shutter 3 **FGR233** (this one hasn't the physical button).

If it is possible to do it without a scenario for me that would be perfect! Otherwise I would gladly take some of your help.

Thank you very much for your next help !

**Comment by AlCalzone** [comment, marked as answer]
The manual of the `MH-C421` should tell you which association groups it has and what they are meant for. If there is one that sends Multilevel Switch or Basic commands, you should be able to use that one and add the FGR223 to it.

**Reply by S1M8N** [reply]
Thank for your quick reply

For my MH-C421 shutter, I have that in manual :
https://i.imgur.com/HlG7YdG.png

But in the configuration I have juste that : 
https://i.imgur.com/NDjS7vh.png

**Reply by AlCalzone** [reply]
> I have that in manual :

That looks like it only sends reports, not controlling commands.

> But in the configuration I have juste that :

That would be the correct one, but it only allows one target node which must be the controller. So I guess you're out of luck here.

**Reply by S1M8N** [reply]
Thank you, So there is no solution ?

**Reply by AlCalzone** [reply]
Seems so 🙁
